ctapipe/io/unofficial/targetio/camera.py

```diff
@@ -0,0 +1,195 @@
+"""
+Module for handling the camera specific information (such as pixel positions),
+which differ between CHEC-S, CHEC-M and single module files.
+"""
+import numpy as np
+from astropy import log
+from os.path import dirname, realpath, join
+from target_io import T_SAMPLES_PER_WAVEFORM_BLOCK as N_BLOCKSAMPLES
+from target_calib import MappingModule, MappingCHEC
+
+
+class Borg:
+    _shared_state = {}
+
+    def __init__(self):
+        """
+        A class used for inheritence to allow "global" configuration classes.
+        """
+        self.__dict__ = self._shared_state
+
+
+class Config(Borg):
+    default = 'checs'
+    # default = 'checm'
+
+    def __init__(self, camera_id=None):
+        """
+        Class for handling the camera specific information (such as pixel
+        positions), which differ between CHEC-S, CHEC-M and single module
+        files.
+
+        The correct configuration can be set globally by importing this class
+        and initializing it as the desired `camera_id`.
+
+        >>> from targetpipe.io.camera import Config
+        >>> Config('checm')
+
+        If a particular script needs a specific configuration other than the
+        default ('checs'), then the above two lines should be the first
+        lines in the script.
+
+        Thanks to the `Borg` inheritence, this class is global, and any new
+        initialization will use the pre-existing configuration, unless a new
+        configuration is chosen.
+
+        Parameters
+        ----------
+        camera_id
+        """
+        Borg.__init__(self)
+        if not self.__dict__:
+            self._id = None
+
+            self.dir = dirname(realpath(__file__))
```

Comment:
I think this will not work, when the package is installed.

Reply:
It looks in the same directory as the python file for the numpy file containing the pixel positions. I think that does work when it is installed, as it also works when looking for the header file for c scripts such as `ctapipe.utils.neighbour_sum`.

Reply:
To my understanding none of the package data files will be installed and thus also will not be found, usually people use `pkg_resources` to find package data.
c.f. http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files